agent/config/config.go

```diff
@@ -570,6 +570,8 @@ func environmentConfig() (Config, error) {
 		SpotInstanceDrainingEnabled:         parseBooleanDefaultFalseConfig("ECS_ENABLE_SPOT_INSTANCE_DRAINING"),
 		GMSACapable:                         parseGMSACapability(),
 		VolumePluginCapabilities:            parseVolumePluginCapabilities(),
+		//TODO: [ecs-exec] Document this new env var to the README file
+		ExecAgentLogDir: os.Getenv("ECS_EXEC_AGENT_LOG_DIR"),
```

Comment:
I don't think this configurable log dir is in the current design.  Is this immediately necessary?  
We'll need to be sure the agent has permissions to access user-defined host level directories.
And, we need to decide what to call it since we're not calling this the `exec_agent` and `ECS_EXECUTE_COMMAND_AGENT_LOG_DIR` is a lot to type.